external.d.ts

```diff
@@ -0,0 +1,11 @@
+declare module "universal-user-agent" {
+    function getUserAgentNode(): string
+    export = getUserAgentNode;
```

Comment:
Can we move this to the top followed by an empty line to match the stile of the block below?